src/components/TextArea.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const TextArea = ({...props}) => (
```

Comment:
Same as previous comment

Reply:
It doesn't work when I change it to just props.